packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -536,7 +533,10 @@ export class ChannelCollection implements IFluidDataStoreChannel, IDisposable {
 			internalId,
 			this.parentContext.IFluidHandleContext,
 		);
-		this.parentContext.addedGCOutboundReference?.(this.containerRuntimeHandle, handle);
+		this.parentContext.addedGCOutboundRoute(
+			this.containerRuntimeHandle.absolutePath,
```

Comment:
This might also be a source of gcUnknownOutboundReferences? Probably not though since nothing changed here, I think.  But when I was working on auto-recovery I was telling GC about new outbound routes from <root> to the thing, but those routes weren't real and didn't show up in the GC data (if it was actually referenced, the real reference from <some actual node> to the thing was there but couldn't map to the fake added route).

In this case though, maybe we're ok - does ContainerRuntime's getGCData include routes from ContainerRuntime -> Aliased things?

Reply:
> But when I was working on auto-recovery I was telling GC about new outbound routes from to the thing, but those routes weren't real and didn't show up in the GC data (if it was actually referenced, the real reference from to the thing was there but couldn't map to the fake added route).

Not sure I follow what you mean here.

Reply:
> In this case though, maybe we're ok - does ContainerRuntime's getGCData include routes from ContainerRuntime -> Aliased things?

I double checked the code. getGCData does include these routes.